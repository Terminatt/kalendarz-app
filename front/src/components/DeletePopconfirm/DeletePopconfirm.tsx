import { Popconfirm, PopconfirmProps } from 'antd';
import React from 'react';

const DeletePopconfirm: React.FC<PopconfirmProps> = (props) => {
    const { children, okButtonProps, cancelButtonProps } = props;

    return (
        <Popconfirm
            {...props}
            okButtonProps={{ ...okButtonProps, className: 'custom-btn custom-btn-delete' }}
            cancelButtonProps={{ ...cancelButtonProps, className: 'custom-btn custom-btn-primary' }}
        >
            {children}
        </Popconfirm>
    );
};

export default DeletePopconfirm;
