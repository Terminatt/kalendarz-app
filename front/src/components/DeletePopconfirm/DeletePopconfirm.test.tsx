import React from 'react';
import renderer from 'react-test-renderer';
import DeletePopconfirm from './DeletePopconfirm';

describe('Delete Popconfirm Component', () => {
    it('matches the snapshot', () => {
        const tree = renderer
            .create(
                <DeletePopconfirm title="test">
                    <button>Test</button>
                </DeletePopconfirm>
            )
            .toJSON();
        
        expect(tree).toMatchSnapshot();
    });
});
