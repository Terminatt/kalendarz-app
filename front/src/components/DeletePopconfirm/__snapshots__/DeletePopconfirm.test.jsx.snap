// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Delete Popconfirm Component matches the snapshot 1`] = `
<button
  onClick={[Function]}
  onKeyDown={[Function]}
  onMouseDown={[Function]}
  onTouchStart={[Function]}
>
  Test
</button>
`;
